src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -510,9 +524,8 @@ private String resolveContent(String result, Set<String> usedIds) {
             return Optional.of(EntryUtil.capitalizeFirst(entry.getType()));
         }
 
-        // TODO: Changed this to also consider alias fields, which is the expected
+        // Changed this to also consider alias fields, which is the expected
```

Comment:
I coulnd't find any, so I thought this could be deleted, but I'm not 100% sure about that. Should I rather keep the comment?